chopsticks-sample/src/main/scala/dev/chopsticks/sample/app/GrpcAkkaMasterApp.scala

```diff
@@ -0,0 +1,147 @@
+package dev.chopsticks.sample.app
+
+import java.util.concurrent.atomic.LongAdder
+
+import akka.actor.ActorSystem
+import akka.stream.KillSwitches
+import akka.stream.scaladsl.{Keep, Sink, Source}
+import com.typesafe.config.Config
+import dev.chopsticks.dstream.DstreamState.WorkResult
+import dev.chopsticks.dstream.Dstreams.DstreamServerConfig
+import dev.chopsticks.dstream._
+import dev.chopsticks.fp.AppLayer.AppEnv
+import dev.chopsticks.fp.DiEnv.{DiModule, LiveDiEnv}
+import dev.chopsticks.fp.akka_env.AkkaEnv
+import dev.chopsticks.fp.iz_logging.IzLogging
+import dev.chopsticks.fp.zio_ext.{MeasuredLogging, _}
+import dev.chopsticks.fp.{AkkaApp, AkkaDiApp, AppLayer, DiEnv, DiLayers}
+import dev.chopsticks.metric.prom.PromMetrics
+import dev.chopsticks.metric.{MetricCounter, MetricGauge}
+import dev.chopsticks.sample.app.proto.grpc_akka_master_worker._
+import dev.chopsticks.stream.ZAkkaStreams
+import io.prometheus.client.{Counter, Gauge}
+import zio._
+
+import scala.concurrent.duration._
+import scala.jdk.DurationConverters.ScalaDurationOps
+
+object GrpcAkkaMasterApp extends AkkaDiApp[Unit] {
```

Comment:
Let's name this `DstreamsSampleMasterApp` instead, probably also put it in its own `dev.chopsticks.sample.app.dstreams` package together with the worker one

Reply:
Also we can move the other `DstreamSampleApp` in there too. Later we'll organize the other FDB sample apps the same way.

Reply:
Done